src/com/homework/lesson7/PetAnimal.java

```diff
@@ -0,0 +1,63 @@
+package com.homework.lesson7;
+
+public class PetAnimal extends Animal {
+    @Override
+    String getFavoriteMeal() {
+        return "Молоко";
+    }
+}
+
+class Cat extends PetAnimal {
```

Comment:
i think will be better to use other packages for pet and wild animals instead of general file